bin/run-test

```diff
@@ -26,4 +26,4 @@ mkdir -p coverage
 docker run --rm \
 	-v $(pwd)/test-reports:/home/tests/reports \
 	-v $(pwd)/coverage:/home/tests/coverage \
-	harmonyservices/harmony-gdal-adapter-test "$@"
+	nasa/harmony-gdal-adapter-test "$@"
```

Comment:
The good thing is that we know the `bin/build-test` and `bin/run-test` changes are working because they are being used in the workflow. So I feel pretty good about the changes to these scripts.